test/e2e-cluster/rolling_restart_recovery_test.go

```
@@ -16,55 +18,124 @@ func TestCleanBootstrapRollingRestartRestoresFullISR(t *testing.T) {
+		rollingRestartISRStabilityWindow,
+		func() (bool, string, error) { return fullISRAndZeroUnderReplicated(ctx) },
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func fullISRAndZeroUnderReplicated(ctx *ClusterTestContext) (bool, string, error) {
+	response, err := ctx.GetClient().SendCommand("", fmt.Sprintf("DESCRIBE topic=%s", ctx.GetTopic()), 5*time.Second)
+	if err != nil {
+		return false, "DESCRIBE failed", nil
```

golangci-lint (2.13.2)

[error] error is not nil (line 115) but it returns nil

(nilerr)